andela/coding-geeks-ah-frontend
# #167484543 finished setting up react router

#### What does this PR do?
It configures **Authors Haven** to use **React Router**.

#### Description of Task to be done?
1. Installing required dependency - react-router-dom.
2. Adding components for testing the react-router functionality.
3. Updating the root component to implement the react-router.

#### How should this be manually tested?
1. Clone this branch.
2. Do **npm install**.
3. Do *npm start** and you should see the landing page.
4. Click on the **About** navigation link and check if the body content changes and click on the **Home** navigation link or the  **AH** logo and check if the body content changes as well.

#### Screenshots
<img width="1440" alt="Screen Shot 2019-09-24 at 17 23 16" src="https://user-images.githubusercontent.com/34879554/65525650-27a2aa80-def0-11e9-8c16-7899be86b1e0.png">
<img width="1439" alt="Screen Shot 2019-09-24 at 17 23 32" src="https://user-images.githubusercontent.com/34879554/65525651-283b4100-def0-11e9-9e27-a9e5d556addf.png">
<img width="1439" alt="Screen Shot 2019-09-24 at 17 27 07" src="https://user-images.githubusercontent.com/34879554/65525944-9122b900-def0-11e9-9c87-dfa5b6938565.png">
